CHANGES.md

```diff
@@ -62,6 +62,12 @@
 
 <!-- Changes that improve Black's performance. -->
 
+### Testing
+
+<!-- Changes that improve Black's testing process. -->
+
+- Simple cases now read automatically from specified subdirectory (#3034)
+
```

Comment:
Do we need this? it's not user facing ...

Reply:
The PR won't pass the CI process if this is missing.
If you want, and you will force-push the PR

Reply:
We can add a label to skip that check. I think it's fine to keep the news entry though, so we give credit to your contribution.